src/components/molecules/CheckedResourcesActionsMenu/CheckedResourcesActionsMenu.tsx

```diff
@@ -66,10 +66,10 @@ const CheckedResourcesActionsMenu: React.FC = () => {
   const dispatch = useAppDispatch();
 
   const checkedResourceIds = useAppSelector(state => state.main.checkedResourceIds);
-  const currentContext = useAppSelector(state => state.config.kubeConfig.currentContext);
   const isInClusterMode = useAppSelector(isInClusterModeSelector);
   const isInPreviewMode = useAppSelector(isInPreviewModeSelector);
   const resourceMap = useAppSelector(state => state.main.resourceMap);
+  const currentConfig = useAppSelector(currentConfigSelector);
```

Comment:
similar to rootFileEntry, if you only need the currentContext, it's better to select just that